Rakefile

```diff
@@ -3,7 +3,9 @@
 
 require File.expand_path('../config/application', __FILE__)
 require 'rspec/core/rake_task'
+require 'rubocop'
```

Comment:
If you're going to require `rubocop` then you can do:

```ruby
require 'rubocop/rake_task'
RuboCop::RakeTask.new(:rubocop)
```

Again, unrelated change should go into a separate PR.